low-level/framework/messaging/ingress_processor.py

```diff
@@ -185,31 +185,44 @@ def _process_msg(self, body):
                     "IngressProcessor, Authentication failed on message: %s" % ingressMsg)
                 return
 
+            # Check for debugging being activated in the message header
+            self._check_debug(message)
+            self._log_debug("_process_msg, ingressMsg: %s" % ingressMsg)
+
             # Get the incoming message type
             if message.get("actuator_request_type") is not None:
                 msgType = message.get("actuator_request_type")
 
                 # Validate against the actuator schema
                 validate(ingressMsg, self._actuator_schema)
+                # Compare hostname from the request to determine
+                # if request is meant for the current node
+                target_node_id = message.get("target_node_id")
+                if target_node_id is None:
+                    logger.warning(
```

Comment:
logger.warning(
                        "Required attribute target_node_id is missing "
                        "from actuator request with %s, IGNORING!!".format(request))

request needs to be deterministic from logs also

Reply:
Done

Reply:
I have removed %s and with from the statement. In egress processer, when request message is sent, entire message is printed in log at debug level, so kept level debug here as well. 
And if `target_node_id` is missing, it should be a warning.

Reply:
Added attributes from `actuator_request_type` in warning message.